docroot/modules/humsci/hs_basic_page/hs_basic_page.install

```diff
@@ -191,3 +191,38 @@ function hs_basic_page_update_9003() {
     }
   }
 }
+
+/**
+ * Update config to hide group heading and filter for paragraph brower widget.
+ */
+function hs_basic_page_update_10000() {
+  $themes_to_update = [
+    'humsci_colorful',
+    'humsci_traditional',
+  ];
+  // Install only on the specified themes.
+  $site_theme = \Drupal::theme()->getActiveTheme()->getName();
+  if (!in_array($site_theme, $themes_to_update, TRUE)) {
+    return;
+  }
+
+  $pages = [
+    'node.hs_basic_page.default' => 'field_hs_page_components',
+    'node.hs_basic_page.default' => 'field_hs_page_hero',
```

Comment:
@hectorlj I think it's better to avoid having duplicated keys in the array. It might work for this particular case, but unexpected things can happen. Maybe it'd be better to have a nested array and then use 2 `foreach` for updating the fields:

```
$pages = [
  'node.hs_basic_page.default' => ['field_hs_page_components', 'field_hs_page_hero'],
  'node.hs_private_page.default' => ['field_hs_priv_page_components'],
];
```

Reply:
Sorry about that! Fixed 👍🏼